packages/esm-form-entry-app/src/app/form-creation/form-creation.service.ts

```diff
@@ -1,13 +1,14 @@
 import { DataSources, EncounterAdapter, Form, FormFactory, PatientIdentifierAdapter } from '@openmrs/ngx-formentry';
 import { Injectable } from '@angular/core';
-import * as moment from 'moment';
+import dayjs from 'dayjs';
```

Comment:
Two things and I'm completely happy with this:

1. Can we just use moment for this and drop dayjs, at least until we wrench moment from the engine.
2. I think the import for moment needs to remain `import * as moment from 'moment'`.